configs/katello/4.1.yaml

```diff
@@ -0,0 +1,102 @@
+---
+:project: katello
+:github_org: katello
+:releases:
+  :4.1.0:
+    :redmine_version_id: 1351
+:prior_releases:
+  :4.0.1:
+    :redmine_version_id: 1369
+  :4.0.0:
+    :redmine_version_id: 1177
+  :3.18.4:
+    :redmine_version_id: 1396
+  :3.18.3:
+    :redmine_version_id: 1370
+:code_name:
+:mash_scripts:
+  - katello-4.1-rhel7
+  - katello-4.1-thirdparty-candlepin-rhel7
+  - katello-4.1-el8
+  - katello-candlepin-4.1-el8
+:repos:
+  :katello:
+    :main: true
+    :branch: KATELLO-4.1
+    :repo: https://github.com/Katello/katello.git
+    :version_branch: true
+  :foreman-packaging:
+    :branch: rpm/2.5
+    :repo: https://github.com/theforeman/foreman-packaging.git
+  :hammer-cli-katello:
+    :branch: 1.1.0
+    :repo: https://github.com/Katello/hammer-cli-katello.git
+  :katello-host-tools:
+    :branch: 3.5.4
+    :repo: https://github.com/Katello/katello-host-tools.git
+  :katello-certs-tools:
+    :branch: 2.7.3
+    :repo: https://github.com/Katello/katello-certs-tools.git
+  :foreman_virt_who_configure:
+    :branch: 0.5.6
+    :repo: https://github.com/theforeman/foreman_virt_who_configure.git
+:ignores:
+:strict_keys: true
+:gpg_key: AC2AD9F1
```

Comment:
gpg key should mirror the corresponding foreman release if it doesn't already

Reply:
thanks! Do you mean one I would find in `configs/foreman/2.5.yaml`? I don't see that that file exists yet..

Reply:
Yep that's where it would be

Reply:
@jturel Got the new GPG key from https://github.com/zjhuntin/theforeman.org/blob/dbc92cd9e81f5be1d5e066514a371f7ca7c0d6a4/static/keys/7EF7E6EA4022F241BB89A149955C283E7522CC3F.pub

(it appears we just use the last 8 chars of the filename for the yaml value here)
Still no `2.5.yaml` in this repo, but I added the GPG key based on the above; hopefully it's correct

Reply:
Seems correct!